src/training/simple.py

```diff
@@ -48,7 +51,7 @@ def train(self):
             self.optimizer.step()
 
             if self.scheduler is not None:
-                self.scheduler.step()
+                self.scheduler.step(loss.item())
```

Comment:
Why do we pass `loss.item()` here?

Reply:
Because some schedulers require losses to determine whether they should modify learning rate on this iteration. Howerver, if the scheduler doesn't need this, then it just ignores the loss IIRC.